src/conditionals/simple_programs_with_decision_points/hard/q1/AbsolutelySureTest.java

```diff
@@ -0,0 +1,32 @@
+package conditionals.simple_programs_with_decision_points.hard.q1;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AbsolutelySureTest extends BaseTest {
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("Enter a number: "),
+                new NewLine(),
+                new IntegerLiteral(5, "returnedValue")
```

Comment:
same here you can assign 5 to its own variable and input it here which makes it less hardcoded